deploy/deploy_db_terminal.yml

```diff
@@ -8,38 +8,38 @@
     - {role: puppet_disable, when: "disable_puppet"}
     - {role: iptables, when: "iptables_config"}
     - common
-    - {role: epel, when: "use_epel and not 'production' in group_names"}
+    - {role: epel7, when: "use_epel and not 'production' in group_names"}
     - umask
-    - {role: log-courier, when: "install_logstash"}
     - jdk
     - postgresql-client
     - eod
-    - devtools
-    - {role: python, when: "custom_repo"}
-    - {role: python-build, when: "not custom_repo"}
-    - python36-scl
-    - python-libs
+    - devtools7
+    - python2rh7
+    - python3rh7
+    - python-libs7
     - sqitch
     - {role: pem-client, tags: pem-client}
     - sublime
-    - odbc
+    - odbc7
   tasks:
 
     - name: Install needed packages
       yum:
-        name: "{{ item }}"
+        name: 
+          - htop
+          - screen
+          - rsync
+          - unzip
+          - gpg
+          - pigz
+          - facter
+          - sysstat
+          - systemtap
+          - ant # sar, iostat, vmstat
+          - p7zip
+          - p7zip-plugins
+          - tesseract
```

Comment:
Can you please add `libpng12` right before `tesseract`?  It is a dependency.  We might want to make a separate role for Tesseract at some point, as we'll need it on Mesos as well.